### PR TITLE
feat: add `assetsDriver` to the RcFile properties

### PR DESCRIPTION
**Pull request by Julien-R44**
Related to https://github.com/adonisjs/assembler/pull/56#discussion_r1002854296

I have not included automatic detection here. I would say that it should be done in the assembler but tell me if I am wrong

### CODE REVIEW

**Comment by RomainLanz** [comment]
Should we use the name `manager` instead of `driver`?
This is the term we are using in the documentation https://docs.adonisjs.com/guides/assets-manager

**Comment by Julien-R44** [comment]
To me, we are defining a "driver for the Assets Manager". It seems to make sense to me
The same logic can be found in config/app.ts

image

A "driver" for the "Assets Manager"

But I don't mind at all if we change it. Let's see what Harminder says

File: adonis-typings/application.ts [hunk 1]
```diff
@@ -136,6 +137,8 @@ declare module '@ioc:Adonis/Core/Application' {
     raw: any
   }
 
+  export type AssetsDriver = 'vite' | 'encore'
```
Comment (thetutlage) [inline adonis-typings/application.ts:140]: I think we should also allow for the `fake` asset driver here? Because that is what gets used during testing

Comment (Julien-R44) [reply adonis-typings/application.ts]: Oops. done

**Pull request merged**